docs/node_management_status.md

```diff
@@ -0,0 +1,117 @@
+# Node Management Status
+
+## Overview
+The Open Horizon policy based, autonomous node management capability is described [here](./node_management.md). 
+
+A Node Management Policy (NMP) status object is generated by the node management worker during node management jobs in order to track the state of the job.
+
+## Definition
+Following are the fields in the JSON representation of an NMP status:
+
+* `agentUpgradePolicyStatus`: A JSON structure to define the status of an automatic agent upgrade job.
+    * `scheduledTime`: An RFC3339 formatted timestamp for when the NMP should start execution.
+    * `startTime`: An RFC3339 formatted timestamp for when the NMP was *actually* executed. 
+    * `endTime`: An RFC3339 formatted timestamp for when the NMP was executed successfully. This field will not be populated if the NMP fails.
+    * `upgradedVersions`: A JSON structure to define the versions being upgraded or downgraded to on a device case. Cluster agents will not use this section.
+        * `softwareVersion`: The version of the agent software packages to be installed.
+        * `certVersion`: The version of the certificate to be installed.
+        * `configVersion`: The version of the config to be installed.
+    * `status`: The state, of the upgrade job. See the section **Status Values** below for more information.
+    * `k8s`: A JSON structure to further define the upgrade progress of a cluster agent.
```

Comment:
Is the k8s section  in the output of `hzn exchange node mgmt status <node>` ? Or is it only in the agent API output?